DZ-8/src/main/java/org/example/StudentsGroup.java

```diff
@@ -0,0 +1,98 @@
+package org.example;
+import java.util.List;
+
+public class StudentsGroup {
+    // Дані про групу, private
+    private Student leader; // Староста
+    private List<Student> students; // Список студентів
+    private List<String> tasks; // Список завдань
+
```

Comment:
Add structure like `Map<String, List<Student>> compleatedTasks = new HashMap<>();` to mark taks as compleated for students